addons/binding/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/protocol/NikoHomeControlDiscover.java

```diff
@@ -31,13 +33,38 @@
  *
  * @author Mark Herwege - Initial Contribution
  */
+/**
+ * @author Mark Herwege - Initial Contribution
+ *
+ */
+@NonNullByDefault
 public final class NikoHomeControlDiscover {
 
     private final Logger logger = LoggerFactory.getLogger(NikoHomeControlDiscover.class);
 
     private InetAddress addr;
-    private String nhcBridgeId;
+    private String nhcBridgeId = "";
+    private boolean isNhcII;
+
+    private static final char[] HEX_ARRAY = "0123456789ABCDEF".toCharArray();
 
+    static String bytesToHex(byte[] bytes) {
```

Comment:
There is a method in openHAB core `HexUtils.bytesToHex` that does the same thing I think.

Reply:
Done.